org.gnome.PowerStats.json

```diff
@@ -0,0 +1,68 @@
+{
+  "app-id": "org.gnome.PowerStats",
+  "sdk": "org.gnome.Sdk",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+  "command": "gnome-power-statistics",
+  "finish-args": [
+    "--share=ipc",
```

Comment:
For X11 you should add `--socket=x11`.
The `--socket=wayland` would be needed if you are interested in native Wayland support as well.

See also:
http://docs.flatpak.org/en/latest/sandbox-permissions.html#standard-permissions

Reply:
Added, thanks.